src/main/java/org/cactoos/text/TextOfString.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2017-2020 Yegor Bugayenko
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included
+ * in all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package org.cactoos.text;
+
+import java.nio.charset.Charset;
+
+/**
+ * Text of {@link String}
+ *
+ * <p>There is no thread-safety guarantee.
+ *
+ * @since 0.49
+ */
+public final class TextOfString extends TextEnvelope {
+
+    /**
+     * Ctor.
+     *
+     * @param input The String
+     * @param cset The Charset
+     */
+    public TextOfString(final String input, final Charset cset) {
+        super(new TextOfScalar(() -> new String(input.getBytes(cset), cset)));
```

Comment:
@baudoliver7 this is quite strange, isn't this equivalent to just use the String itself? I would propose to rewrite this class by directly implementing `Text` instead of delegating to `TextOfScalar` and to remove the `Charset` which makes no sense to me.